### ISSUE TITLE
0.20.194 : ArrayIndexOutOfBoundsException 

### ISSUE THREAD

**Issue by Mopster** [Haehnchen/idea-php-symfony2-plugin#1476]
I updated the plugin to the latest version a couple of minutes ago and I get the following error. PhpStorm is no longer analyzing the code.

PhpStorm 2020.1.1
Build #PS-201.7223.96, built on April 30, 2020
Runtime version: 11.0.6+8-b765.40 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.15.0-101-generic
GC: ParNew, ConcurrentMarkSweep
Registry: run.processes.with.pty=TRUE
Non-Bundled Plugins: de.espend.idea.laravel, de.espend.idea.php.annotation, fr.adrienbrault.idea.symfony2plugin

```
java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at fr.adrienbrault.idea.symfony2plugin.doctrine.ObjectManagerFindContextTypeProvider.complete(ObjectManagerFindContextTypeProvider.java:81)
	at com.jetbrains.php.PhpIndexImpl.lambda$completeType$1(PhpIndexImpl.java:129)
	at com.intellij.openapi.util.RecursionManager$1.doPreventingRecursion(RecursionManager.java:117)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:129)
	at com.jetbrains.php.PhpIndexImpl.getClasses(PhpIndexImpl.java:708)
	at com.jetbrains.php.PhpIndexImpl.getBySignatureInternal(PhpIndexImpl.java:617)
	at com.jetbrains.php.PhpIndexImpl.getBySignature(PhpIndexImpl.java:508)
	at com.jetbrains.php.PhpIndexImpl.getBySignature(PhpIndexImpl.java:496)
	at com.jetbrains.php.lang.inspections.exception.PhpExceptionAnalysisService.getBySignature(PhpExceptionAnalysisService.java:316)
	at com.jetbrains.php.lang.inspections.exception.PhpExceptionAnalysisService.collectThrownExceptions(PhpExceptionAnalysisService.java:270)
	at com.jetbrains.php.lang.inspections.exception.PhpExceptionAnalysisService.lambda$getPreventingRecursion$4(PhpExceptionAnalysisService.java:243)
	at com.intellij.openapi.util.RecursionManager$1.doPreventingRecursion(RecursionManager.java:117)
	at com.jetbrains.php.lang.inspections.exception.PhpExceptionAnalysisService.getPreventingRecursion(PhpExceptionAnalysisService.java:240)
	at com.jetbrains.php.lang.inspections.exception.PhpExceptionAnalysisService.lambda$getThrownExceptions$2(PhpExceptionAnalysisService.java:213)
	at com.intellij.psi.impl.PsiCachedValueImpl.doCompute(PsiCachedValueImpl.java:54)
	at com.intellij.util.CachedValueBase.lambda$getValueWithLock$1(CachedValueBase.java:235)
	at com.intellij.openapi.util.RecursionManager$1.doPreventingRecursion(RecursionManager.java:117)
	at com.intellij.openapi.util.RecursionManager.doPreventingRecursion(RecursionManager.java:75)
	at com.intellij.util.CachedValueBase.getValueWithLock(CachedValueBase.java:236)
	at com.intellij.psi.impl.PsiCachedValueImpl.getValue(PsiCachedValueImpl.java:43)
	at com.intellij.util.CachedValuesManagerImpl.getCachedValue(CachedValuesManagerImpl.java:73)
	at com.jetbrains.php.lang.inspections.exception.PhpExceptionAnalysisService.getThrownExceptions(PhpExceptionAnalysisService.java:212)
	at com.jetbrains.php.lang.inspections.exception.PhpExceptionAnalysisService.getThrownExceptions(PhpExceptionAnalysisService.java:199)
	at com.jetbrains.php.lang.inspections.PhpThrownExceptionsAnalyzer.getExceptionClasses(PhpThrownExceptionsAnalyzer.java:75)
	at com.jetbrains.php.lang.inspections.PhpThrownExceptionsAnalyzer.getExceptionClasses(PhpThrownExceptionsAnalyzer.java:65)
	at com.jetbrains.php.lang.inspections.phpdoc.PhpDocRedundantThrowsInspection$buildVisitor$1.getThrownExceptions(PhpDocRedundantThrowsInspection.kt:77)
	at com.jetbrains.php.lang.inspections.phpdoc.PhpDocRedundantThrowsInspection$buildVisitor$1.visitPhpFunction(PhpDocRedundantThrowsInspection.kt:46)
	at com.jetbrains.php.lang.inspections.phpdoc.PhpDocRedundantThrowsInspection$buildVisitor$1.visitPhpMethod(PhpDocRedundantThrowsInspection.kt:40)
	at com.jetbrains.php.lang.psi.elements.impl.MethodImpl.accept(MethodImpl.java:100)
	at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:69)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$null$6(LocalInspectionsPass.java:320)
	at com.intellij.util.AstLoadingFilter.lambda$toComputable$2(AstLoadingFilter.java:168)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:126)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:115)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:110)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitRestElementsAndCleanup$7(LocalInspectionsPass.java:320)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:149)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:162)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1101)
	at com.intellij.concurrency.ApplierCompleter.lambda$wrapInReadActionAndIndicator$1(ApplierCompleter.java:105)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:627)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:572)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:61)
	at com.intellij.concurrency.ApplierCompleter.wrapInReadActionAndIndicator(ApplierCompleter.java:117)
	at com.intellij.concurrency.ApplierCompleter.lambda$compute$0(ApplierCompleter.java:96)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:168)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:168)
	at com.intellij.concurrency.ApplierCompleter.compute(ApplierCompleter.java:96)
	at com.intellij.concurrency.JobLauncherImpl.invokeConcurrentlyUnderProgress(JobLauncherImpl.java:55)
	at com.intellij.concurrency.JobLauncher.invokeConcurrentlyUnderProgress(JobLauncher.java:49)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.visitRestElementsAndCleanup(LocalInspectionsPass.java:331)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.inspect(LocalInspectionsPass.java:197)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.collectInformationWithProgress(LocalInspectionsPass.java:115)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:84)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:52)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$null$1(PassExecutorService.java:442)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1106)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:435)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:627)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:572)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:61)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:434)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:410)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:168)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:168)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:408)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:171)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
```

I reverted back to the previous version 0.19.193

**Comment by maxtorete** [comment]
Same here. I tried the "Invalidate cache & Restart" trick described in README, but no luck there either.

**Comment by joel-ns** [comment]
Seems fixed in 0.20.195. Thanks @Haehnchen!
